docs/source/events/index.rst

```diff
@@ -54,6 +54,7 @@ All events are encoded as JSON. On top of the JSON built-in types the events can
 - date: ISO 8601 encoded dates either with an explicit offset or in UTC
```

Comment:
Why might it be a) with an explicit offset, or b) in UTC? Any reason why it might vary? I think people will want to know which to expect.

Could also give an example so people don't have to look up ISO 8601

Reply:
I have added examples.

Treating a missing offset as ±00:00 shouldn’t be hard to implement. Ensuring either a) or b) in IST 1.0 and IST 2.0 is around 1d of work.